src/players/services/players.service.ts

```diff
@@ -173,19 +169,15 @@ export class PlayersService implements OnModuleInit {
   async forceCreatePlayer(
     playerData: ForceCreatePlayerOptions,
   ): Promise<Player> {
-    let etf2lProfile: Etf2lProfile;
-    try {
-      etf2lProfile = await this.etf2lProfileService.fetchPlayerInfo(
-        playerData.steamId,
-      );
-    } catch (error) {
-      etf2lProfile = undefined;
-    }
+    const etf2lProfile =
+      (await this.etf2lProfileService.fetchPlayerInfo(playerData.steamId)) ??
```

Comment:
`Etf2lProfileService.fetchPlayerInfo()` might throw an error in case there is no such player, this should be handled.